src/index.js

```diff
@@ -60,6 +61,12 @@ module.exports = {
    *
    * @see {@link https://github.com/adobe/aio-lib-customer-profile/blob/master/README.md|@adobe/aio-lib-customer-profile}
    */
-  CustomerProfile
+  CustomerProfile,
+  /**
+  * The Adobe Experience Platform Real-Time Customer Profile module of the Adobe I/O SDK
```

Comment:
This documentation is wrong. I'm confused how the README.md has this correct, but in the source it is wrong, since the README should be generated from the jsdoc via `npm run docs`.

Reply:
Thanks shaz
resolved, I have pushed the last commit.